tests/unit/src/test/scala/tests/ScalaToplevelSuite.scala

```diff
@@ -99,7 +102,7 @@ class ScalaToplevelSuite extends BaseSuite {
     List(
       "_empty_/A.", "_empty_/A.foo().", "_empty_/A.Z#", "_empty_/B#",
       "_empty_/B#X#", "_empty_/B#foo().", "_empty_/C#", "_empty_/D#",
-      "_empty_/D.Da.", "_empty_/D.Db.",
+      "_empty_/D.Da. -> _empty_/D#", "_empty_/D.Db. -> _empty_/D#",
```

Comment:
Actually, should we show those if there is no extends clause, do the cases by default extend the enum?

Reply:
I think that's just syntax sugar, and
```
enum Color:
  case Red
``` 
and
```
enum Color:
  case Red extends Color
``` 
mean the same thing.